src/main/java/org/example/PESEL.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+public class PESEL {
+
+    public static boolean isStringType(String numberPesel) {
+        try {
+            if (numberPesel instanceof String) {
```

Comment:
Jesli parametr wejściowy jest typu String to on zawsze będzie instanceof String. Powinnaś tutaj sprawdzic czy zmienna numberPesel posiada w sobie tylko znaki liczb.